db/migrate/20160210013622_create_wikis.rb

```diff
@@ -0,0 +1,33 @@
+class CreateWikis < ActiveRecord::Migration
+  def change
+    create_table :wikis do |t|
+      t.string :language, limit: 16
+      t.string :project, limit: 16
+    end
+
+    default_wiki = Wiki.find_or_create_by(
+      language: ENV['wiki_language'],
+      project: 'wikipedia'
+    )
+
+    add_column :articles, :wiki_id, :integer, index: true
+    add_column :assignments, :wiki_id, :integer, index: true
+    add_column :revisions, :wiki_id, :integer, index: true
+
+    reversible do |dir|
```

Comment:
So, I went ahead and used the models for all of my migrations.  However, I found this recently:
http://guides.rubyonrails.org/v3.2.8/migrations.html#using-models-in-your-migrations

The argument is that models will eventually diverge enough that they cannot be used to manipulate older schema.  Seems pretty compelling...